tests/rs_sqla_test_utils/utils.py

```diff
@@ -5,8 +5,9 @@
 
 
 def clean(query):
-    encodings_removed = re.sub(r'\s+ENCODE\s+\w+', '', query)
-    return re.sub(r'\s+', ' ', encodings_removed).strip()
+    query = re.sub(r'\s+ENCODE\s+\w+', '', query)
+    query = re.sub(r'\s+CONSTRAINT\s+[a-zA-Z0-9_".]+', '', query)
```

Comment:
This scrubs out all `CONSTRAINT constraintname` strings out of DDL when we're comparing, and we're able to avoid changing the expected DDL we code in the tests.